app/views/plans/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for(@plan) do |f| %>
+<%= form_for(@plan, html: { 'data-parsley-validate': '' }) do |f| %>
```

Comment:
normally, there is no need to add pre and post spacing in html files for such tags.